era5cli/utils.py

```diff
@@ -186,4 +186,4 @@ def _append_netcdf_history(ncfile: str, appendtxt: str):
             {}'''.format(appendtxt, ncfile.history))
     except AttributeError:
         ncfile.history = appendtxt
-    ncfile.close()
+    ncfile.close()
```

Comment:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Please use flake8 to see whether the code adheres to the PEP8 python style guidelines.